documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

```diff
@@ -194,6 +194,13 @@ For your convenience, there are two procedures to create migration plans, starti
 * To start with a source provider, see xref:creating-migration-plan-2-6-3_provider[Creating a migration plan starting with a source provider].
 * To start with specific VMs, see xref:creating-migration-plan-2-6-3_vms[Creating a migration plan starting with specific VMs].
 
+[WARNING]
+====
+The administrator must ensure that no data consistency issues arise when a guest initiates storage connections independently, such as Internet Small Computer Systems Interface (iSCSI) connections or Network File System (NFS) mounts.
```

Comment:
Virtual machines with guest initiated storage connections, such as Internet Small Computer Systems Interface (iSCSI) connections or Network File System (NFS) mounts, are not handled by {product} and might require additional planning before or reconfiguration after the migration.

Reply:
How about this?

Reply:
Grand so. Updated. Thanks